src/Sulu/Component/Content/Document/Subscriber/WorkflowStageSubscriber.php

```diff
@@ -270,19 +273,19 @@ private function setWorkflowStage(
      * @param NodeInterface $node
      * @param string $locale
      * @param int $workflowStage
-     * @param bool $updatePublished
+     * @param \DateTime $publishDate
      */
-    private function setWorkflowStageOnNode(NodeInterface $node, $locale, $workflowStage, $updatePublished)
+    private function setWorkflowStageOnNode(NodeInterface $node, $locale, $workflowStage, $publishDate)
```

Comment:
You've only fixed 2 of 3 calls of this method, was this by accident, or can the call in `setWorkflowStageToTestForRestore` really pass `false`? Would prefer `null` and a type hint here, if we have to change it like this.